net-vpn/i2pd/i2pd-2.28.0.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd cmake-utils toolchain-funcs
```

Comment:
We usually leave one empty line below EAPI to follow default skel type, remember next time.

Reply:
ok!